.github/workflows/build.yml

```diff
@@ -264,22 +264,30 @@ jobs:
         run: npm ci
         working-directory: ./src/vscode-bicep
         
-      - name: Compile
-        run: npm run compile
-        working-directory: ./src/vscode-bicep
-
       - name: Run lint
         run: npm run lint
         working-directory: ./src/vscode-bicep
+          
+      - name: Run unit tests
+        run: npm run test:unit
+        working-directory: ./src/vscode-bicep
+
+      - name: Run snapshot tests
+        run: npm run test:snapshot
+        working-directory: ./src/vscode-bicep
```

Comment:
Unit tests and snapshot tests are using `ts-jest` now which doesn't require running `tsc` first.